test/gpu/native/studies/transpose/transpose-throughput.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: Performance (GB/s):, Performance (GB/s):
```

Comment:
Just as a piece of information, you can have a `.graph` file describe multiple graphs. `find test -name "*graph" | xargs -L1 grep -H perfkeys | cut -d":" -f1 | uniq -c | sort -rn | head` shows examples of this.

What do I prefer? I don't know. :) I think I prefer less effort, so keep this as is. And use your judgement in the next one?

I kind of like seeing one line per plot in the GRAPHFILES. But for plots like this, it might make sense to have a single `.graph` file? I don't know.

Reply:
That's good to know, thanks!